build.assets/tooling/cmd/build-os-package-repos/config.go

```diff
@@ -137,6 +138,7 @@ func NewConfigWithFlagset(fs *flag.FlagSet) *Config {
 		}
 	})
 	fs.StringVar(&c.releaseChannel, "release-channel", "", "The release channel of the repos that the artifacts should be added to")
+	fs.BoolVar(&c.targetCloud, "target-cloud", false, "True to target a \"cloud\" release channel, false otherewise")
```

Comment:
Why did you decide to use the boolean "cloud" flag, instead of just having the caller of the tool explicitly provide "cloud" as a channel flag value?

Reply:
I can do that - does a `--channel-override` flag work? Currently "channel" is determined by the artifact major version.

Reply:
That would work better for me, or perhaps "subchannel" since the full channel seems to be `stable/X`.

You could also propagate that term into the deb repo stuff which has a `majorVersion` field. I was initially a little perplexed by setting the major version to "cloud" as this is not valid semver. If it were called (sub)channel throughout, it would make more sense; that the channel comes from the major version mostly, but can be overridden is pretty clear.